sdk/docs/manually-written/sdk/sdlc-howtos/smart-contracts/develop/patterns/implementing-time-constraints.rst

```diff
@@ -0,0 +1,22 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _implementing-time-constraints:
+
+How To Implement Time Constraints
+#################################
+
+Restricting Choices to Occur After a Fixed Period of Time
+*********************************************************
+
+TODO: https://github.com/DACH-NY/docs-website/issues/315
+
+Restricting Choices to Occur Before a Deadline
```

Comment:
Use sentence case: https://docs.google.com/document/d/153D6-gI0blsAvlrzReHVyz8RCHUQzzdKnxnpcADlt6c/edit?tab=t.0#bookmark=id.ggvywsrwxaoe

Reply:
in all titles